### ISSUE TITLE
[DevNote] Update Requests library to version 2.0.8

### ISSUE THREAD

**Issue by abhansnuk** [WordPress/Documentation-Issue-Tracker#1152]
Trac ticket: https://core.trac.wordpress.org/ticket/59322

Labeled 'Field Guide'

Owner of ticket: costdev

**Comment by nalininsbs** [comment]
Unclear on the follow up for this. @abhansnuk can you take a look as need for FG.

**Comment by abhansnuk** [comment]
Yes I was looking at this too. I have sent a [message](https://wordpress.slack.com/archives/C02RQBWTW/p1697545374244079) in Core. It does not need a dev note too, just a call out in the FG.
